docs/conf.py

```diff
@@ -62,9 +62,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.0'
+version = '1.1'
 # The full version, including alpha/beta/rc tags.
-release = '1.0.0'
+release = '1.1.0'
```

Comment:
Yeah, that could be helpful for sure. We really should be able to get to a fully scripted release process. All we need is someone to volunteer to be the official Falcon release manager. 😉